pkts-sip/src/main/java/io/pkts/packet/sip/impl/SipMessageBuilder.java

```diff
@@ -717,10 +730,12 @@ public T build() {
         msg.write(SipParser.CR);
         msg.write(SipParser.LF);
 
-        for (final SipHeader header : finalHeaders) {
-            header.getBytes(msg);
-            msg.write(SipParser.CR);
-            msg.write(SipParser.LF);
+        for (final List<SipHeader> headerValues : finalHeaders.values()) {
+            for (final SipHeader header : headerValues) {
+                header.getBytes(msg);
+                msg.write(SipParser.CR);
+                msg.write(SipParser.LF);
+            }
```

Comment:
In your benchmarking tests, do you have a test for constructing sip messages as well? I can imagine this double-for-loop may slow things down, question is if it matters. If you could check in your benchmarking somewhere so we can expand on that as well, that would be great.